src/tablebuilder.ts

```diff
@@ -9,13 +9,13 @@ function formattedErrorFor(benchmark: Benchmark): string {
 }
 
 export function buildBenchmarkTable(benchmarks: Benchmark[]): string {
-    var tableString = "| Benchmark | Score | Error |\n" +
-                      "| --------- | ----- | ----- |\n";
-
+    let tableString = `| Benchmark | Score | Error |`
+    tableString += `\n`
+    tableString += `| --------- | ----- | ----- |`
+    tableString += `\n`
     benchmarks.forEach(function (benchmark) {
-        tableString +=
-            "| " + benchmark.benchmark + " | " + formattedScoreFor(benchmark) + " | " + formattedErrorFor(benchmark) + " |\n"
-    });
-
-    return tableString;
+        tableString += `| ${benchmark.benchmark} | ${formattedScoreFor(benchmark)} | ${formattedErrorFor(benchmark)} |`
+        tableString += `\n`
+    })
+    return tableString.trimEnd()
```

Comment:
I guess this `trimEnd` is here to get rid of any trailing `\n`? I think if we rearrange this a bit then we don't need to trim the end
```ts
    let tableString = `| Benchmark | Score | Error |`
    tableString += `\n`
    tableString += `| --------- | ----- | ----- |`
    benchmarks.forEach(function (benchmark) {
        tableString += `\n`
        tableString += `| ${benchmark.benchmark} | ${formattedScoreFor(benchmark)} | ${formattedErrorFor(benchmark)} |`
    })
    return tableString
    ```

Reply:
Good idea 👌